### ISSUE TITLE
[Launch] BDD: Create Test Users in Dev and Staging for BDD

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11892]
## Issue Description
Create dev and staging users for testing BDD.

Dev and Staging Users:
1. User 1: Military History that will have a RAD date 90-180 days out.
2. User 2: Military History that will have a RAD date 0-89 days out.

## Acceptance Criteria
- [ ] Create two dev users.
- [ ] Create two staging users.
---
## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by annaswims** [comment]
We'll be fine without Service history from eMIS.  The military history prefill code should be unchanged from what's in prod for All Claims. We can just add a  service record with a future date when we fill the BDD form.

**Comment by annaswims** [comment]
in dev: need users with: edipi, ssn, birls_id, participant_id
search for `^PI^200BRLS^USVBA^A` in [](https://github.com/department-of-veterans-affairs/vets-api-mockdata/tree/master/mvi) to find users with a BIRLS id.

`^PI^200CORP^USVBA^` to find participant_id

**Comment by annaswims** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/pull/151

Big list of claims users:
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users-birls-participant-edipi.csv

**Comment by annaswims** [comment]
I've correlated these logins with what what Michael Harlowe messaged me.

-----

vets.gov.user+247@gmail.com - Nicholas Willis has a dozen pending claims and I cancelled them all, so he's a good candidate for BDD testing.

vets.gov.user+250@gmail.com - I submitted a BDD for Clifford Wilson through Swagger and it worked. I cancelled it so he's ready to test from staging.

vets.gov.user+258@gmail.com
vets.gov.user+261@gmail.com 
Jerry Wright and ALANA GPRSRESYSFIVE have no claims.
